### DISCUSSION TITLE
High compute time on Postgres

### DISCUSSION THREAD

**Discussion by hartmann-jonas** [vercel/community#2374] [Help] [answered]
### Summary

I am experiencing crazy high compute times on my Vercel Postgres database using the hobby plan.
As far as I understood the compute time is calculated by multiplying the active hours of the database by the number of CPUs., therefore my calculation for a daily usage with 24 active hours should be 6h of compute time (24*0.25 = 6).

Still I have usages of over 7 hours with basically no traffic on the page.
Am I missing something?

**Comment by amyegan** [comment, marked as answer]
Hi @hartmann-jonas! Usage reporting was slightly delayed on May 1, so some of your May 1 usage may have been reported as May 2 usage. If the higher usage day was a different date, please you share a screenshot of your Postgres compute time graph to help us look into the usage reporting bug 🙏

**Reply by hartmann-jonas** [reply]
Here's my graph

<img width="943" alt="image" src="https://github.com/vercel/community/assets/80170229/bf8c3615-c603-453a-9d1b-0968f5aeb31b">

On some of those days with almost 6h of compute time the site had no visitors at all and therefore the database should not be able to reach numbers this high

Edit: I just checked the dashboard again and the compute time now shows a reasonable value. So seems to be fixed. Thanks!
<img width="935" alt="image" src="https://github.com/vercel/community/assets/80170229/aa7504ec-1e9d-4228-8d22-caa1051da9c1">

**Reply by amyegan** [reply]
Thanks for reporting the problem, and for coming back to share that it's working appropriately for you now 😄

**Comment by sebous** [comment]
Hello, I'm seeing simillar issue on our project. We're on the PRO plan and my test database created on 11.5 had used 1.56 compute hours already. I've inserted few rows to one table, the number is way off in my opinion.  We want to use the vercel postgres for internal tool, but this would go out of hand very quickly. Thanks for any help on this 🙌 

![image](https://github.com/vercel/community/assets/14926669/ba9f1ebe-4104-4b24-9f7c-dab078f40260)
![image](https://github.com/vercel/community/assets/14926669/d274491d-c7d4-4f5e-a3e9-2d218ea046ed)

**Comment by hfoley2013** [comment]
I am also having extremely high compute time. It looks like my connection is never going dead. My charts are showing 24 hours of compute time every single day. This is despite the fact that the website has no traffic and data revalidation is set to two week periods. 

@amyegan  Any insights into why this might be the case? This has been happening on hobby and pro plans. 

Happy to walk through the repo and see if there is something glaring that I am missing.
![image](https://github.com/vercel/community/assets/109574453/9fb8ccca-51a2-486d-a930-0d51bf357c95)

**Reply by amyegan** [reply]
Which database should I investigate? If you already have a [support case](https://vercel.com/help#issues) open, you can share the name/ID there and let me know the case number. 🙏

**Reply by hfoley2013** [reply]
I do not have a support case open, but here is the db endpoint: ep-solitary-morning-29344794-pooler.us-west-2.postgres.vercel-storage.com

If I need to open a support ticket, please let me know. I'm now up to almost 250 hours of compute and cannot for the life of me see what's going wrong.
